### DISCUSSION TITLE
第6节课笔记(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#470] [笔记]
第6节课笔记(13班)

**Comment by qingyin741** [comment]
https://zhuanlan.zhihu.com/p/679168242

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/FQ1GwPEIhigehFkyY0Nc0bv8nxb?from=from_copylink

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135857262
